modules/kmm-unloading-kernel-module.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="kmm-unloading-kernel-module_{context}"]
+= Unloading the kernel module
+
+.Procedure
+
+* To unload a module loaded with KMM from nodes, delete the corresponding `Module` resource. KMM then creates worker pods where required to run `modprobe -r` and unload the kernel module from nodes.
```

Comment:
Reword?

Reply:
Change "KMM then creates worker pods where required to run `modprobe -r` and unload the kernel module from nodes." to "KMM then creates worker pods, where required, to run `modprobe -r` and unload the kernel module from nodes."?

Reply:
I am good with that change.